Dockerfile

```diff
@@ -32,18 +32,17 @@
 
 FROM python:3.12-slim AS base
 
-ARG DOCKER_BUILDKIT
-
 WORKDIR /app
 
 # Install common OS-level dependencies
-# TODO: when moving to ONS infrastructure, replace RUN with
+# TODO: when moving to ONS infrastructure, replace RUN
```

Comment:
Was this intentional?

Reply:
it was, but on reflection I should revert it